ckanext/opendatani/plugin.py

```diff
@@ -208,6 +219,34 @@ def report_resources_by_organization(context, data_dict):
     return resource_data
 
 
+@toolkit.side_effect_free
+def prepare_csv_report(context, data_dict):
```

Comment:
Why this action is in `plugins.py`? Actions belong in `actions.py` directory

Also, I'm just thinking - why do we need this as an action? More seems like a helper function. It creates a file and returns its name and only. Don't think we need to have it accessible via API. Let's move it to helpers